package.json

```diff
@@ -4,13 +4,13 @@
   "description": "Make travel and accommodation easy and convenient.",
   "main": "index.js",
   "scripts": {
-    "test": "NODE_ENV=test npm run dropTables && NODE_ENV=test sequelize db:migrate && NODE_ENV=test sequelize db:seed:all && NODE_ENV=test mocha --require @babel/polyfill --require @babel/register ./src/tests/mainTest.test.js",
+    "test": "NODE_ENV=test npm run dropTables && NODE_ENV=test sequelize db:migrate && NODE_ENV=test sequelize db:seed:all && NODE_ENV=test mocha --require @babel/polyfill --require @babel/register ./src/tests/mainTest.test.js --timeout 60000 --exit",
     "coverage": "nyc npm test && nyc report --reporter=text-lcov | coveralls",
     "dropTables": "sequelize db:migrate:undo:all",
     "createTables": "sequelize db:migrate && sequelize db:seed:all",
     "build": "babel src --out-dir dist --source-maps inline --copy-files",
-    "start": "nodemon dist/index.js",
-    "dev-start": "nodemon --exec babel-node src/index.js"
+    "start": "nodemon dist/app.js",
+    "dev-start": "nodemon --exec babel-node src/app.js"
```

Comment:
It seems like code in index.js was moved to app.js and hence changed the `dev-start` script.
@Kagorora , is there any reason?

Reply:
Yes, I did change our entry point, from index.js to app.js  as  I needed socket.io to listen on the same port as  our server, as a consequence I have changed our script to start the server and deleted index.js as it was no longer used anywhere,